### DISCUSSION TITLE
Unsure where to submit but want to bring to GitHub's attention - received suspicious email from 'notifications@github.com'

### DISCUSSION THREAD

**Discussion by reddtoric** [community/community#84704] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I've looked and I'm unsure where else to report this.

I received a phishing email but was from notifications@github.com and signed by github. I've never starred that repo before. I've already reported the email as phishing (to Google).

![p2](https://github.com/community/community/assets/14900721/b3ac7e1d-6f9b-46cc-b80e-39ce2f720607)
![details](https://github.com/community/community/assets/14900721/ccc060df-98fa-4795-ad2a-8ca7697283e7)

**Comment by okplanbo** [comment]
Hi @reddtoric, Got similar email today, looks like TAO token scam went through gmail anti-spam 
![image](https://github.com/community/community/assets/26038255/099ea3ae-ddf5-4558-a2c5-7b5bb4c27bcb)

Clearly phishing as link hrefs are different from URL in link text.

G'day @Iron-Ham,
help is needed to reach someone from github security team.

Regards,
Boris

**Comment by karasowles** [comment]
Thanks for reporting this @reddtoric & @okplanbo.

@billythekid's response on a similar phishing attempt applies here as well:
https://github.com/orgs/community/discussions/81478#discussioncomment-8113627
